lib/eex/lib/eex/engine.ex

```diff
@@ -98,21 +101,24 @@ defmodule EEx.Engine do
   @doc """
   Returns an empty string as initial buffer.
   """
-  def init(_opts) do
+  @spec init(Keyword.t) :: <<>>
+  def init(opts) when is_list(opts) do
```

Comment:
I don't feel like it is adding useful information. Specially given how straight-forward the implementation is. It is equivalent to spec'ing private functions. Sure some could find it more useful but we believe it is generally not worth it. My main motivation for using specs is for documentation and here it won't make a difference.